exporter/zipkinexporter/config.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zipkinexporter
+
+import (
+	"github.com/open-telemetry/opentelemetry-service/config/configmodels"
+)
+
+// Config defines configuration settings for the Zipkin exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	// Endpoint is the URL to send the Zipkin trace data to (e.g.:
+	// http://some.url:9411/api/v2/spans).
+	Endpoint string `mapstructure:"endpoint"`
```

Comment:
Question: is `Endpoint` a good choice for this? I wonder if it can be confusing that sometimes Endpoint is just address:port and sometimes an url.

Reply:
That is a good point. `http-url` since it is sends to HTTP servers... let me know if you have any other suggestions. I will update the readme from other PR accordingly.

Reply:
Going with `http-url` for yaml and `HTTPAddress` for the field name, `HTTPURL` will be requested by lint and it doesn't look nice

Reply:
How about simply `url`?

Reply:
In principle it must be HTTP since Zipkin will send a HTTP POST request. If there is consensus around it I won't object to `url`.

Reply:
While url indeed can be non-http, http is so ubiquitous now that I think it is always assumed to be the default when one mentions a url. I think the `http-` will be unnecessary, it does not add much value for end user. `url` seems to be a good enough hint to the user to understand what value to put here.

Reply:
Happy to hear other opinions, otherwise the PR looks good to me.

Reply:
ok, let's go with `url`